addons/sale/models/sale.py

```diff
@@ -885,6 +885,23 @@ def _get_payment_type(self):
         self.ensure_one()
         return 'form_save' if self.require_payment else 'form'
 
+    def _get_payment_methods(self):
+        """Returns available payment methods for the sale order."""
+        self.ensure_one()
```

Comment:
again, why this method and not calling the model method on the payment acquirer?